app/gui2/src/components/GraphEditor/clipboard.ts

```diff
@@ -14,107 +21,148 @@ interface ClipboardData {
 /** Node data that is copied to the clipboard. Used for serializing and deserializing the node information. */
 interface CopiedNode {
   expression: string
-  metadata: NodeMetadataFields | undefined
+  documentation?: string | undefined
+  visualization?: VisualizationMetadata | undefined
+  colorOverride?: string | undefined
```

Comment:
It would be nice if, by default, we would copy all the metadata. So we could keep `metadata` field as `Omit<NodeMetadataFields, 'position'>`